src/features/CanvasEditor.tsx

```diff
@@ -210,46 +130,27 @@ const CanvasEditor = () => {
                       backgroundColor: '#edeff5'
                     }}
                   >
-                    {rendering && (
-                      <>
-                        <CanvasContainer
-                          canvasWidth={1090}
-                          // socket={socket}
-                          appDefinition={appDefinition}
-                          appDefinitionChanged={appDefinitionChanged}
-                          snapToGrid={true}
-                          // darkMode={this.props.darkMode}
-                          mode={'edit'}
-                          zoomLevel={zoomLevel}
-                          deviceWindowWidth={100}
-                          selectedComponents={selectedComponents}
-                          // appLoading={isLoading}
-                          // onEvent={handleEvent}
-                          // onComponentOptionChanged={
-                          //   handleOnComponentOptionChanged
-                          // }
-                          // onComponentOptionsChanged={
-                          //   handleOnComponentOptionsChanged
-                          // }
-                          setSelectedComponent={setSelectedComponent}
-                          // handleUndo={handleUndo}
-                          // handleRedo={handleRedo}
-                          // removeComponent={removeComponent}
-                          onComponentClick={handleComponentClick}
-                          onComponentHover={handleComponentHover}
-                          hoveredComponent={hoveredComponent}
-                          // sideBarDebugger={sideBarDebugger}
-                          currentPageId={currentPageID}
-                        />
-                        <CustomDraggerLayer
-                          snapToGrid={true}
-                          canvasWidth={1090}
-                          onDragging={() => {
-                            console.log('dragging');
-                          }}
-                        />
-                      </>
-                    )}
+                    {/* {true && ( */}
+                    <>
+                      <CanvasContainer
+                        canvasWidth={1090}
```

Comment:
try to keep constant as used at multiple places

Reply:
Yes